store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -11269,6 +11269,20 @@ public void refreshUserCredentials(Account account) throws ServiceException {
         return habOrgList;
     }
 
+    @Override
+    public Set<String> listHabOrgUnit(Domain domain) throws ServiceException {
+        Set<String> habOrgList = null;
+        try {
+            habOrgList = getAllHabOrgUnitInADomain(domain);
+        } catch (ServiceException e) {
+            throw ServiceException.FAILURE(
+                String.format("Unable to list HAB org units for domain=%s", domain.getName()), e);
+        } finally {
+            LdapClient.closeContext(zlc);
```

Comment:
store build failed for me at this statement, looks like it needs  below statement at start of method -
ZLdapContext zlc = null;